app/controllers/users_controller.rb

```diff
@@ -221,7 +225,8 @@ def sso_discourse
     all_groups = User.all_discourse_groups
 
     # Get the teams/councils/Delegate status for user
-    user_groups = current_user.current_teams.select(&:official_or_council?).map(&:friendly_id)
+    user_groups = current_user.current_teams.select(&:official?).map(&:friendly_id)
+    user_groups.concat(current_user.active_roles.map { |role| UserRole.discourse_user_group(role) }.compact)
```

Comment:
`concat` changes the array that it's being called on under the hood, even though this is breaking Ruby's own naming conventions (it should be `foo.concat!` to make clear that `foo` is being modified in-place).

I would be happier with something like `user_groups +=` or `user_groups |=` instead.

Reply:
Okay, changed to `+=`